src/tls/test-server.c

```diff
@@ -21,6 +21,7 @@
 
 #include <errno.h>
 #include <fcntl.h>
+#include <glob.h>
```

Comment:
We don't need globbing any more, so this can go.

Reply:
Good catch, thanks.